API/Controllers/UserController.cs

```diff
@@ -111,22 +115,27 @@ public async Task<IActionResult> GetUser(int userId)
 
 
         /// <summary>
-        ///     Create a user account.
+        /// Creates the account asynchronous.
         /// </summary>
-        /// <param name="accountResource"></param>
+        /// <param name="accountResource">The account resource.</param>
         /// <returns></returns>
```

Comment:
Empty returns tag (while this method does return something).
Please, also fix the other empty returns tags in this file :)